src/Custom/Responses/Streaming/StreamingResponseImageGenerationCallPartialImageUpdate.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace OpenAI.Responses;
+﻿using System;
+
+namespace OpenAI.Responses;
 
 // CUSTOM: Renamed.
 [CodeGenType("ResponseImageGenCallPartialImageEvent")]
 public partial class StreamingResponseImageGenerationCallPartialImageUpdate
 {
+    // CUSTOM: Renamed.
+    [CodeGenMember("PartialImageB64")]
```

Comment:
We should fix this in the client.tsp by adding a JSON usage as discussed offline.